Sources/Redis/RedisID.swift

```diff
@@ -1,31 +1,72 @@
 import Foundation
 import Vapor
 
-public struct RedisID: Hashable, Codable {
+/// A type-safe representation of a String representing individual identifiers of separate Redis connections and configurations.
+///
+/// It is recommended to define static extensions for your definitions to make it easier at call sites to reference them.
+///
+/// For example:
+/// ```swift
+/// extension RedisID { static let oceanic = RedisID("oceanic") }
+/// app.redis(.oceanic) // Application.Redis instance
+/// ```
+public struct RedisID: Hashable,
+                       Codable,
+                       RawRepresentable,
+                       ExpressibleByStringLiteral,
+                       ExpressibleByStringInterpolation,
+                       CustomStringConvertible,
+                       Comparable {
 
-    public let string: String
-    public init(string: String) {
-        self.string = string
+    public let rawValue: String
+
+    public init(stringLiteral: String) {
+        self.rawValue = stringLiteral
+    }
+
+    public init(rawValue: String) {
+        self.rawValue = rawValue
     }
 
-    public static let `default` = RedisID(string: "default")
+    public init(_ string: String){
+        self.rawValue = string
+    }
+
+    public typealias RawValue = String
+    public typealias StringLiteralType = String
```

Comment:
You don't need to change this, but just an FYI that the compiler can infer this when you just provide the `init(rawValue:)` and `init(stringLiteral:)` implementations, as well as the `rawValue: String` definition

Reply:
haha yeah I prefer the compiler inferring it. Didn't know which you would prefer.